src/pc/djui/djui_chat_box.c

```diff
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include "pc/network/network.h"
 #include "pc/lua/smlua_hooks.h"
```

Comment:
you need to include djui_interactable.h or else there wont be any SCANCODE_ stuff

Reply:
what
if you don't include it then all the SCANCODE_ stuff will be NULL, and so all of it will pretty much be 0